apps/fleet-desktop-macos/dev-notify-smoke.sh

```
@@ -0,0 +1,53 @@
+#!/bin/bash
+# Assert the notify exit-code contract.
+#
+# Only covers cases that exit before AppKit starts, so it needs no display and no
+# logged-in user — which makes it safe to run in CI. The paths that actually put a
+# window on screen have to be checked by hand; see dev-notify.sh.
+
+set -uo pipefail
+
+cd "$(dirname "$0")"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)